### ISSUE TITLE
Link to nested topic broken in HTML5 output

### ISSUE THREAD

**Issue by eerohele** [dita-ot/dita-ot#2443]
`develop`.

[Files](https://github.com/eerohele/dita-ot-issues/tree/master/fixtures/2443).

This:

``` xml
<!-- topic1.dita -->
<xref href="topic2.dita#topic2-nested"/>

<!-- topic2.dita -->
<topic id="topic2" xml:lang="en-us">
  ...

  <topic id="topic2-nested" xml:lang="en-us"> ... </topic>

</topic>
```

Results in:

``` html
<!-- topic1.html -->
<a class="xref" href="topic2.html#topic2-nested"> ... </a>

<!-- topic2.html -->
<article id="topic2__topic2-nested"> ... </article>
```

There's a mismatch between the `@href` target ID and the actual target ID.

**Comment by mironovalexey** [comment]
Related links:
https://github.com/dita-ot/dita-ot/pull/2276
http://docs.oasis-open.org/dita/dita/v1.3/os/part3-all-inclusive/archSpec/base/id.html#id

It seems that the href attribute has the correct value, but the id of the article doesn't.

Furthermore
the `<xref href="innertopic_id/section_id"/>`
and the appropriate
`<topic id="topic_id"><topic id="innertopic_id"><section id="section_id">...</></></>`
leads to the correct result
`<a href="innertopic_id__section_id">...</a>...<section id="innertopic_id__section_id">.`

Guess we don't need to [call the gen-toc-id template here](https://github.com/dita-ot/dita-ot/blob/develop/src/main/plugins/org.dita.html5/xsl/topic.xsl#L258):
```
  <!--<xsl:call-template name="setidaname"/>-->
  <xsl:attribute name="id" select="@id"/>
```

**Comment by robander** [comment]
Fixed with #2686.
